### DISCUSSION TITLE
Is StableSwarmUI only for nvidia GPU?

### DISCUSSION THREAD

**Discussion by pinea00** [Stability-AI/StableSwarmUI#94] [Q&A]
AMD GPU is not work~~

**Comment by mcmonkey4eva** [comment]
Swarm itself works anywhere, as only the backend has to work on your GPU. You can technically even eg install swarm on a PC with no GPU and then set the backend to a remote service like StabilityAPI or a Colab instance of ComfyUI.

Nvidia is the main focus (as nvidia just kinda works better for AI in general), but there are ways to get it to work on Nvidia, AMD. Mac, even on CPUs (slowly)

For AMD specifically, see here: https://github.com/Stability-AI/StableSwarmUI/issues/23#issuecomment-1667174967

> - Open `StableSwarmUI\dlbackend\comfy\ComfyUI`
> - open a command line there
> - run `..\python_embeded\python.exe -m pip install torch-directml`

(The linked issue is a standing feature request to make this work automatically without you having to manually run commands, which should be done before Swarm is out of Alpha status)
